components/account/UserSets.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <b-container data-qa="user sets">
```

Comment:
This data-qa attribute is overwritten when the component is used in the page.
It's also not used for any tests as far as I see, so I don't think it's needed.